README.md

```diff
@@ -14,7 +14,7 @@ This plugin uses local graphviz dot executable for rendering.
 
 
 ## Usage
-1. Install graphviz if you haven't yet.
+1. Install graphviz if you haven't yet or use the experimental d3-graphviz support.
```

Comment:
1. Could we add more details like with "d3-graphviz" you don't need to install dot on your local and it will work for mobile?
1. Could we add a contributor section to include all committers to thank their contributions? Thanks.

Reply:
I don't think it's needed but, feel free to update and extend the documentation, it is your plugin ;-)